doc/source/modules.rst

```diff
@@ -14,3 +14,4 @@ Modules
    networks
    pipes
    utilities
+   properties
```

Comment:
An alternative to `properties` could be `fluid` (it contains properties for fluids) or `media` (if it ever grows to include soil/grout as well).

Reply:
`media` is fine with me. I'd prefer that to `fluid` for if we do get around to including other objects.